t/btc.t

```diff
@@ -0,0 +1,452 @@
+    'previousblockhash' => '000000000005a678d87340aa0e21704eb5d0673d442a7ab0725bd169dff76897',
+    'nonce'             => 2125810177,
+    'version'           => 536870912,
+    'merkleroot'        => 'b3b0bc9d339815842493796724dfc984eefbf6c122dba421c506f485c5ee20f7',
+    'nTx'               => 1,
+    'bits'              => '1d00ffff',
+    'weight'            => 896,
+    'strippedsize'      => 215,
+    'tx'                => [{
+            'weight' => 572,
+            'hash'   => 'a2494b617bbae185645f9678a5bffd77b011f35aa66aff0c9fa7bfb6763fd378',
+            'size'   => 170,
+            'hex' =>
+                '010000000001010000000000000000000000000000000000000000000000000000000000000000ffffffff0403b06c19ffffffff02e40b54020000000017a914c61461766896a2becc3a2bbd82369c46b7ef4b24870000000000000000266a24aa21a9ede2f61c3f71d1defd3fa999dfa36953755c690689799962b48bebd836974e8cf90120000000000000000000000000000000000000000000000000000000000000000000000000',
+            'version'  => 1,
+            'vsize'    => 143,
+            'locktime' => 0,
+            'vout'     => [{
+                    'scriptPubKey' => {
+                        'addresses' => ['2NCJunLYyxigRUQqVYMSdAfKh5zMmvZ9CYW'],
+                        'hex'       => 'a914c61461766896a2becc3a2bbd82369c46b7ef4b2487',
+                        'asm'       => 'OP_HASH160 c61461766896a2becc3a2bbd82369c46b7ef4b24 OP_EQUAL',
+                        'reqSigs'   => 1,
+                        'type'      => 'scripthash'
+                    },
+                    'n'     => 0,
+                    'value' => '0.01050212'
+                }
+            ],
+            'txid' => 'e7cc12f01de0860e867043ea877744f989e6f6d769c4cb5004c5a2475cc7c393',
+            'vin'  => [{
+                    'sequence' => 4294967295,
+                    'coinbase' => '03b06c19'
+                }]}
+    ],
+    'nextblockhash' => '0000000023563d72e89c7e185d90e153739667b413ee0b99c83002b941da8663',
+    'versionHex'    => '20000000',
+    'size'          => 251,
+    'height'        => 1666224,
+    'mediantime'    => 1582098354,
+    'chainwork'     => '00000000000000000000000000000000000000000000014280a400c353418ad4'
+};
+my $get_transaction_value = {
+    'amount'             => '0.01050212',
+    'blocktime'          => 1582002572,
+    'walletconflicts'    => [],
+    'bip125-replaceable' => 'no',
+    'blockindex'         => 37,
+    'confirmations'      => 138,
+    'details'            => [{
+            'category' => 'receive',
+            'address'  => '2NCJunLYyxigRUQqVYMSdAfKh5zMmvZ9CYW',
+            'amount'   => '0.01050212',
+            'label'    => 'CR90000002',
+            'vout'     => 0
+        }
+    ],
+    'time'         => 1582001792,
+    'blockhash'    => '000000000038c85a491a62fff3257f02a57c571d10fbe6740d0fe4a6921461fc',
+    'txid'         => 'e7cc12f01de0860e867043ea877744f989e6f6d769c4cb5004c5a2475cc7c393',
```

Comment:
Can this very long string use a variable ? that will make it more clear.

Reply:
I'm not sure if this will make a difference, as you see there are a lot of the long data.
I tried to not repeat the test data as you can see in the subtests and just overwrite the needed part only.

Reply:
I think it does make sense. For exmaple, [line 88](https://github.com/binary-com/perl-Net-Async-Blockchain/pull/31/files/0d52cb2663eee15af685e28c22073cfda45ca985#diff-6cff0d5044be9c0eb17e6eff175452a8R88) and [line 96](https://github.com/binary-com/perl-Net-Async-Blockchain/pull/31/files/0d52cb2663eee15af685e28c22073cfda45ca985#diff-6cff0d5044be9c0eb17e6eff175452a8R96) is the same string, But the reader cannot get this point at the first glance.  And [this line](https://github.com/binary-com/perl-Net-Async-Blockchain/pull/31/files/0d52cb2663eee15af685e28c22073cfda45ca985#diff-6cff0d5044be9c0eb17e6eff175452a8R143) is used in 4 places.